packages/template-retail-react-app/app/components/display-price/index.jsx

```diff
@@ -40,8 +40,8 @@ const DisplayPrice = ({
                 </Text>
             )}
             <Text
-                as={discountPrice > 0 ? 's' : 'b'}
-                ml={discountPrice > 0 ? 2 : 0}
+                as={typeof discountPrice === 'number' ? 's' : 'b'}
```

Comment:
a dumb question, what is `s` and `b`?

Reply:
Strikethrough / bold. https://chakra-ui.com/docs/components/text